### DISCUSSION TITLE
Help regarding conversion of .ats files into txt format

### DISCUSSION THREAD

**Discussion by anilkuhembram** [BRGM/razorback#13] [Q&A] [answered]
I have many .ats files recorded from metronix system these files are in binary format, can I use your program to convert them into txt format. If so please suggest.

**Comment by fsmaibrgm** [comment, marked as answer]
```python
import razorback as rzb
import pathlib
p = pathlib.Path()
for ats in p.rglob('*.ats'):
    s = rzb.io.ats.load_ats([str(ats)])
    print(ats, s.sampling_rate, s.start, s.data[0].compute())
```
This will print the ats file name, the sampling rate, the start time and the data values of each ats files in a directory.   
Surely, you can start from that to get what you want.

**Reply by anilkuhembram** [reply]
Thank you so much, sir for your help. From this code, I was able to get the data values.

**Reply by anilkuhembram** [reply]
Sir, I got the data points in text format from the conversion of '.ats' file. But sir I have a query regarding this data.
First, the starting time which is coming after running the program.......what does it denote mean is it the time from where the recording started? If yes, then what will be the stopping time?
Second, if an ats file contains all five bands of MT data (Ex, Ey, Hx, Hy, and Hz), then I want to store them separately in different columns but in a single column.
[109A01AF.txt](https://github.com/BRGM/razorback/files/11553071/109A01AF.txt)

I have attached the file which I have converted through this code.
